lib/image-diff.js

```diff
@@ -102,6 +104,14 @@ ImageDiff.prototype = {
     var diffPath = options.diffImage;
     var shadow = options.shadow;
 
+    if((typeof options.imageMagick === 'undefined') || (options.imageMagick === true)) {
+      gm = require('gm').subClass({imageMagick: true});
```

Comment:
We should now be using `this.gm` instead of `gm` since we create it via the constructor